src/theme/pfas/_colors.scss

```diff
@@ -5,16 +5,24 @@
   color: #F5F6FA !important;
 }
 .gray-0 {
-  color: #ABAEBB !important;
+  color: rgba(0, 0, 0, 0.3) !important;
 }
 .gray-1 {
-  color: #A4A8B7 !important;
+  color: rgba(0, 0, 0, 0.4) !important;
 }
 .gray-2 {
-  color: #818496 !important;
+  color: rgba(0, 0, 0, 0.6) !important;
 }
 .gray-3 {
-  color: #565B70 !important;
+  color: rgba(0, 0, 0, 0.8) !important;
+}
+.black {
+  color: black !important;
+}
+// use in activity, event components
+// for list item shared component for expired event title
+.color-gray-2 {
```

Comment:
no need anymore

Reply:
@shawnm0705 In activity component and in events we pass this 
`[titleColor]="eventListService.isNotActionable(event) ? 'gray-2' : ''"`
and in list item component we put title color like this
`<p class="item-title subtitle-1" [ngClass]="'color-' + titleColor">{{ title }}</p>`

Reply:
that's why i create a class color-gray-2

Reply:
Then we just need to put this class inside the css of list-item component only, it is not used anywhere else